src/core/kube/resource_type.ts

```diff
@@ -24,4 +24,5 @@ export enum ResourceType {
   CLUSTER_ROLE_BINDING = 'ClusterRoleBinding',
   CLUSTER = 'Cluster',
   CONTAINER = 'Container',
+  PVC = 'PVC',
```

Comment:
These should match the Kubernetes `kind:` descriptor.

```suggestion
  PVC = 'PersistentVolumeClaim',
```